src/utils/useGuidelines.ts

```diff
@@ -2,10 +2,9 @@ import { useQuery } from "@tanstack/react-query";
 import axios from "axios";
 import _ from "lodash";
 import { Guideline, ServerGuideline } from "../models/guideline";
-import { valueToLevel } from "./valueToLevel";
 
 const QUERY_KEY = "guidelines";
-const URL = "https://c0ldbear.github.io/data/wsg-info.json";
+const URL = "https://c0ldbear.github.io/data/v1/wsg-info.json";
```

Comment:
FYI: Uploaded the new data to another folder. Will change this if we accept this :)